.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+## Description
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Checklist
+
+- Is this feature complete or do we expect follow-up changes before this code can ship?
+  - [ ] Yes
+  - [ ] No
+- Are you including unit tests for the changes and scenario tests if relevant?
+  - [ ] Yes
+  - [ ] No
+- Did you add public API?
```

Comment:
Can we also ask whether the public API has XML doc comments including:

1. `<remarks />` if greater explanation is required than what can be provided in the summary.
2. `<example />` elements for any examples with code placed in a `<code />` element.